Documentation/configuration.md

```diff
@@ -141,6 +141,12 @@ Be CAUTIOUS to use unsafe flags because it will break the guarantee given by con
 + Force to create a new one-member cluster. It commits configuration changes in force to remove all existing members in the cluster and add itself. It needs to be set to [restore a backup][restore].
 + default: false
 
+### Logging Flags
+
+#### -disable-log-timestamps
```

Comment:
I think this flag is too specific. What about using output type? e.g. http://www.freedesktop.org/software/systemd/man/journalctl.html#-o

My concern is that 1. we may need to modify other parts in the log line in the future for systemd usage 2. for debug usage, we may even need a more precise log format.